public/data/food/ingredients.json

```diff
@@ -6793,8 +6771,8 @@
       "plotSize": 0.0
     },
     "id": "6c3815da-dec2-4cf2-9be6-9d99b4e9948d",
-    "inedible_part": 0,
-    "land_occupation": 1.7603417799441383,
+    "inedible_part": 0.0,
```

Comment:
Would there be any way to prevent zeros to be rendered as `0.0`? Testing for specific `0` values before processing other numbers in the formatter should do the trick in theory, but you're never too sure

Reply:
I’ve commited the fix, could you give it a try?